blockchain/staking/run_a_validator.md

```diff
@@ -30,6 +40,8 @@ The script will ask you for confirmation before every action. It will:
 * configure time synchronization
 * generate session keys for use in next step
 
+The script gives the node 30 seconds to start up, if it takes longer it will exit. If that happens, just wait a minute and run the script again.
```

Comment:
Running again after a minute will skip generating session keys. 
This PR fixes this problem: https://github.com/liberland/liberland_substrate/pull/242/files
Maybe we should expedite merging the fix instead of changing docs?